src/lib/mongoBackend/dbConstants.h

```diff
@@ -120,6 +120,7 @@
 #define CSUB_BLACKLIST               "blacklist"
 #define CSUB_ONLYCHANGED             "onlyChanged"
 #define CSUB_COVERED                 "covered"
+#define CSUB_NOTIFYONMETADATACHANGE  "notifyOnMetadataChange"
```

Comment:
New field should be added to doc/manuals/admin/database_model.md#csubs-collection